articles/article-116-esoteric-programming-languages.md

```diff
@@ -0,0 +1,27 @@
+<!-- BEGIN TITLE -->
+# Esoteric Science
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+There are thousands of programming languages. But some of them are just laborious and inefficient. They are not designed for practical usage but to implement different aspects of unusual language concepts. Those programming languages are called esoteric. One of them is called Brainfuck. It consists of the eight language commands:
+
+<  > + -  . , [ ]
+
+There are more comments needed to explain the code than the size of the code itself.
```

Comment:
Suggestion: Maybe we could write: To be able to understand the code, more comments are needed to explain the code than the size of the code itself.

Reply:
done.